omod/src/main/compass/sass/dateRangePicker.scss

```diff
@@ -1,16 +1,12 @@
-.date-range-picker-div {
-  display: inline-block;
-}
```

Comment:
@sherrif10 , you cleared  this class  but you still reference it in the daterangepicker.html file

Reply:
Added the date-range-picker-div class like you said co-mentor, any thing else

Reply:
How is removing this related to the ticket?

Reply:
I interchanged this code, there was no much logic, the only challenge was misplacing the code.so i only placed where the code need to be to present it much more reasonable. the problem was just designing, but otherwise the same code does the same. let me hear from your reveiw thanks